ui/pages/confirmations/components/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -27,17 +30,27 @@ const AdvancedGasFeeGasLimit = () => {
   const [isEditing, setEditing] = useState(false);
   const [gasLimit, setGasLimit] = useState(gasLimitInTransaction);
   const [gasLimitError, setGasLimitError] = useState();
+  const chainId = useSelector(getCurrentChainId);
 
   const updateGasLimit = (value) => {
     setGasLimit(value);
   };
 
   useEffect(() => {
     setGasLimitInContext(gasLimit);
+    if (chainId === CHAINLIST_CHAIN_IDS_MAP.MANTLE) {
```

Comment:
Is it worth creating an array so we can add to this in future, such as `IGNORE_GAS_LIMIT_CHAIN_IDS`?

Reply:
Done 👍